src/vg/civcraft/mc/citadel/Utility.java

```diff
@@ -878,6 +888,35 @@ public static MultiBlockReinforcement createMultiBlockReinforcement(List<Locatio
         return rein;
     }
     
+    /**
+     * Display an effect defined in the config around a reinforcement.
+     * @param reinforcement The reinforcement to spawn the effect around. 
+     * @return Whether an effect was displayed or not. 
+     */
+	public static boolean playReinforcementEffect(PlayerReinforcement reinforcement) {
+		ReinforcementEffectType reinforcementEffect = ReinforcementType.getReinforcementType(reinforcement.getStackRepresentation()).getReinforcementEffect();
+		if(reinforcementEffect == null){
+			return false;
+		}
+		
+		Effect effect = reinforcementEffect.getEffect();
+		if (effect == null) {
+			return false;
+		}
+		
+		Location centerLocation = reinforcement.getLocation().clone().add(0.5, 0.5, 0.5);
+		for (int i = 0; i < reinforcementEffect.getAmount(); i++) {
+			if (reinforcementEffect.getViewDistance() != 0) {
+				centerLocation.getWorld().playEffect(centerLocation, effect, 0, reinforcementEffect.getViewDistance());
```

Comment:
Use this method (https://hub.spigotmc.org/javadocs/spigot/org/bukkit/entity/Player.Spigot.html#playEffect(org.bukkit.Location,%20org.bukkit.Effect,%20int,%20int,%20float,%20float,%20float,%20float,%20int,%20int)) instead, it puts all particles in one package.